cmd/root.go

```diff
@@ -135,7 +139,12 @@ func getLatestReleaseInfo(info chan<- string) {
 	if err != nil {
 		// The error is intentionally not being handled because we don't want
 		// to stop the execution of the program because of this failure
-		log.Debugf("Error checking if newer odo release is available: %v", err)
+		log.GetInstance().Logf(
```

Comment:
This is kind of ugly, is there a cleaner way of doing this?

Reply:
This looks overcomplicated.
We should be able to do just `glog.V(4).Infof(....`